modelplace_api/utils.py

```diff
@@ -17,6 +17,19 @@
     from ._rle_mask import encode_binary_mask, decode_coco_rle
 
 
+def is_numpy_array_equal(result: np.ndarray, gt: np.ndarray, error: float) -> bool:
+    '''
+    Applies element-wise comparison of two ndarrays and defines
+    whether the ratio of matched elements is eligible
+    '''
+    if result.size != gt.size:
+        raise RuntimeError('"result" and "gt" arrays must have equal size')
+    elements_num = result.size
+    equal_elements_num = np.equal(result, gt).sum()
```

Comment:
I believe it's not suitable for float arrays' comparison. Probably, it's better to use something like np.isclose?

Reply:
Updated, thanks!